README.md

```diff
@@ -156,6 +105,70 @@ Sometimes issues will come up with your Docker instance than cannot be solved wi
     * If you receive the message `"docker rmi" requires at least 1 argument.` it means that there are no downloaded images; it is safe to proceed.
 4. Download images and rebuild containers: `docker-compose up`
 
+### Environment Variables
+
+We use [environment variables](https://medium.com/chingu/an-introduction-to-environment-variables-and-how-to-use-them-f602f66d15fa) to control some configuration aspects of the application. When deployed to Heroku, these values are set using the "config var" apparatus, but in our local development environment, we don't have access to those. In most cases, we rely on defaults in development, but there are a few cases where environment variables are necessary.
+
+We use the `dotenv` [Ruby gem](https://github.com/bkeepers/dotenv) and [npm package](https://www.npmjs.com/package/dotenv). These packages allow us to define environment variables in our local environment using a file called `.env`. For example, the backend requires a variety of secret keys to generate JSON Web Tokens, and we define those in our `.env` file. When the application starts up, it reads those variables from that file.
+
+Since the `.env` file can contain sensitive information, we don't commit it (it's [gitignored](https://git-scm.com/docs/gitignore)), so you need to create one on your local computer. To set yours up, copy the existing `.env.example` file to the `.env` file:
+
+```
+cp .env.example .env
+```
+
+This will provide the minimum you need to run the site on your local computer. If you need to work on signup/confirmation/forgot password workflows, you'll also need to set the sendgrid configuration to enable sending emails. Talk to a tech lead if you need those credentials.
+
+
+### Editorconfig
+
+Please install one of the following code editors (unless you're already using one of them): VSCode, Atom, Sublime Text
+
+We have a `.editorconfig` file which specifies some default text editor settings that should help with code styling (e.g. cleaning up whitespace). This requires a plugin to work, so follow the link below to the install instructions for you specific editor:
+
+* [VSCode](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig)
+* [Atom](https://atom.io/packages/editorconfig)
+* [Sublime Text](https://github.com/sindresorhus/editorconfig-sublime)
+
+
+The local setup ends here. Please continue to read about Steps to Develop Locally, about more tools and useful commands we're using, below:
+_________________________________________________________________________________________________________
+
+## Steps to spin the app locally
+
+1. Clone this repository locally unless you've already done so:
+git clone git@github.com:the-difference-engine/rescue-leftover-cuisine2.git
+
+2. in your terminal, navigate locally to the rescue-leftover-cuisine2 folder
+
+3. run 'bundle install'
+
+4. run 'rails server' or 'rails s' to start the server
+
+5. TBD for frontend (how do we develop frontend code?)
```

Comment:
Unfortunately my personal laptop needs care (some basic tools need to be reinstalled, I haven't done it in 3 years) and I wasn't able to pass the bundle install step.

Would you be able to help me add the proper steps here?

Reply:
Sure! These are the steps to get up and running with the front-end:

```
5. Change your current working directory to the `frontend` folder (`cd frontend` if you're currently in the project root)
6. Run `npm install`
7. Run `PORT=5000 npm start`
8. Navigate to 'http://localhost:5000' in your browser

Reply:
Thanks a lot!